test/execution_test.js

```diff
@@ -1,269 +1,272 @@
-                                    "format":"#,,.00M",
-                                    "identifier":"adyRSiRTdnMD.generated.pop.1fac4f897bbb5994a257cd2c9f0a81a4"
-                                }
-                            }
-                        ];
-                        ex.getData('myFakeProjectId', ['attrId', 'metricId'], {
-                            definitions: definitions
-                        });
-
-                        var request = this.server.requests[0];
-                        var requestBody = JSON.parse(request.requestBody);
-                        expect(requestBody.execution.definitions).to.eql(definitions);
+            describe('getData with execution context filters', function() {
+                it('should propagate execution context filters to the server call', function() {
+                    // prepare filters and then use them with getData
+                    const filters = [{
+                        'uri': '/gdc/md/myFakeProjectId/obj/1',
+                        'constraint': {
+                            'type': 'list',
+                            'elements': ['/gdc/md/myFakeProjectId/obj/1/elements?id=1']
+                        }
+                    }];
+                    ex.getData('myFakeProjectId', ['attrId', 'metricId'], {
+                        filters: filters
                     });
+                    const request = this.server.requests[0];
+                    const requestBody = JSON.parse(request.requestBody);
+
+                    expect(requestBody.execution.filters).to.eql(filters);
                 });
+            });
 
-                describe('getData with query language filters', function() {
-                    it('should propagate filters to the server call', function() {
-                        // prepare filters and then use them with getData
-                        var where = {
-                            'label.attr.city': { '$eq' : 1 }
-                        };
-                        var data = ex.getData('myFakeProjectId', ['attrId', 'metricId'], {
-                            where: where
-                        });
+            describe('getData with order', function() {
+                it('should propagate orderBy to server call', function() {
+                    const orderBy = [
+                        {
+                            column: 'column1',
+                            direction: 'asc'
+                        },
+                        {
+                            column: 'column2',
+                            direction: 'desc'
+                        }
+                    ];
+                    ex.getData('myFakeProjectId', ['attrId', 'metricId'], {
+                        orderBy: orderBy
+                    });
 
-                        var request = this.server.requests[0];
-                        var requestBody = JSON.parse(request.requestBody);
+                    /*eslint-disable vars-on-top*/
```

Comment:
You mean the rule or the case? I kinda like the rule but in this specific case it cannot be on top of the block because call to server has to happen before...